### DISCUSSION TITLE
Kaggle - Introduction to Machine Learning

### DISCUSSION THREAD

**Discussion by Akash-Samanta** [dsc-tiu/MLStudyJams2022#3] [Show and tell]
Hey guys, I am Akash Samanta from GDSC-TIU. Just got my first certificate 🥇  from the ⭐ ML Study Jams 2022 ⭐ ! Here's to learning and gaining many more !

Here's the Repo: https://github.com/naxatra6/Kaggle-Introduction-to-Machine-Learning

![image](https://user-images.githubusercontent.com/65293175/187366661-25ff4343-c132-4184-8e4d-6c56eec301e7.png)

**Comment by diffrxction** [comment]
Well done! Keep up the good work! 👍🏻
